src/libs/Log.ts

```diff
@@ -82,4 +83,21 @@ const Log = new Logger({
 });
 timeout = setTimeout(() => Log.info('Flushing logs older than 10 minutes', true, {}, true), 10 * 60 * 1000);
 
+AppLogs.configureAppGroupName('group.com.expensify.new');
+AppLogs.registerHandler({
+    filter: '[NotificationService]',
+    handler: ({filter, logs}) => {
+        logs.forEach((log) => {
+            // both native and JS logs are captured by the filter so we replace the filter before logging to avoid an infinite loop
```

Comment:
```suggestion
            // Both native and JS logs are captured by the filter so we replace the filter before logging to avoid an infinite loop
```

Keep comments capitalized please

Reply:
@kirillzyusko bump ^

Reply:
Still need to update this as well.